mozilla-services/tecken
# bug 1703253: upgrade to django 3.2

9d2b06e2 bug 1703253: fix frontend and static handling
9c88a667 bug 1703253: change some settings values to lists
00c5d1a1 bug 1703253: switch some views to use FileResponse
301f4eee bug 1703253: fix token API
3a726483 bug 1703253: swap url out for re_path
745a2d09 bug 1703253: remove default_app_config
40e14eb5 bug 1703253: update warnings filter
910203b6 bug 1703253: add DEFAULT_AUTO_FIELD and fix STATIC_URL
5b5d1c63 bug 1703253: remove available_attrs use
5265061b bug 1703253: update Django to 3.2.11 and pyscopg2 to 2.9.3

Test plan:

1. `make build`
2. `make lintfix` and `make lint`
3. `make test`
4. run everything with `make run`
   1. flip through the web app
   2. make sure to log in and flip through the protected views
   3. flip through the django admin
      Note that clicking on "view site" in the admin throws an error, but it did that before, too. It has to do with how the frontend does this weird thing in the local dev environment.
5. Log in, make an API token, and run the system tests against localhost

Then check `http://localhost:8090` to see if anything is in fakesentry.
